### ISSUE TITLE
Feature-Request: Show completions

### ISSUE THREAD

**Issue by Jojaa350** [itchyny/lightline.vim#547]
First of all, thanks for developing this plugin, it's my favorite by far!
There is just one inconvenience which I wish could be resolved:

Regular (clean) vim displays a bar with all suggestions when pressing tab:
![VIM-clean suggestions](https://user-images.githubusercontent.com/39162942/106466135-3c576100-649b-11eb-8256-edd09306cd3f.png)
With lightline, I loose this functionality.

Has anybody considered implementing this feature into lightline?

**Comment by itchyny** [comment]
For me, lightline and wildmenu works well together. The statusline switches to the completion candidates automatically on pressing the tab key. Please share your entire vimrc.

**Comment by Jojaa350** [comment]
OK, so, as it turns out, wildmenu does work, it just was deactivated by default - which is odd, considering that using `vim --clean` has it switched on.
Anyway, I didn't know that this feature was called "wildmenu" in VIM and thus was not able to get the reference from vim's help. I tried googling, but as this feature just seemed to disappear upon using lightline (for whatever reason - I now tried executing my vimrc by hand in `vim --clean` and wildmenu still worked...), I always googled for "lightline tab completion" or similar.

For anybody having the same issue and coming here from Google: `set wildmenu` fixed the problem.

I apologize for bothering you @itchyny, but you already helped me out without knowing, just by asking for my vimrc. 😄
